### DISCUSSION TITLE
account.schedule().list_calendars() shows only 2 out of 3 !?

### DISCUSSION THREAD

**Discussion by ghost** [O365/python-o365#902] [Q&A]
Greetings,

I use an outlook.office.com account to access some other peoples calendars (in the same organisation) . Two of those (test) users have given me access so I can make a calendar event, change it, delete it (using outlook.office.com in a browser, so interactive as a user would do).

So far so good. However, when reading the calendars in Python with account.schedule().list_calendars() only 2 out of 3 are shown. I don't think this is an O365 issue but an Outlook rights issue, but I'm a little puzzled with all those settings?

Any suggestions in the field?

![image](https://user-images.githubusercontent.com/39137738/205646475-79cd063a-92cf-4e83-b4a5-b23a259d5cca.png)

![image](https://user-images.githubusercontent.com/39137738/205646669-ea06dbb1-6c7a-41be-aa1d-c716f86dfe46.png)

[AzureTestCalendarList.zip](https://github.com/O365/python-o365/files/10154458/AzureTestCalendarList.zip)

**Comment by alejcas** [comment]
I don't have any clue.  try to set the resource like:

```python
print(account.schedule('account_xxxx').list_calendars())
```

**Reply by ghost** [reply]
Hello Janscas ,

This is not an O365 issue or an issue of my own code. 
I did investigate this issue with the Microsoft  Graph Explorer and got the same result.
Need to dig into the Outlook and Azure delegate settings and all that kind of stuff.

**Reply by alejcas** [reply]
ok, then this should be something related to permissions
